src/third_party/boost/boost/system/error_code.hpp

```diff
@@ -211,9 +211,6 @@ namespace boost
     inline const error_category &  get_system_category() { return system_category(); }
     inline const error_category &  get_generic_category() { return generic_category(); }
     inline const error_category &  get_posix_category() { return generic_category(); }
-    static const error_category &  posix_category = generic_category();
```

Comment:
In general, we prefer to not change the boost sources unless overwhelmingly required. What warning or error prompted this change?